bot/exts/moderation/modpings.py

```diff
@@ -55,6 +65,53 @@ async def reschedule_roles(self) -> None:
                 expiry = isoparse(pings_off[mod.id]).replace(tzinfo=None)
                 self._role_scheduler.schedule_at(expiry, mod.id, self.reapply_role(mod))
 
+    async def reschedule_modpings_schedule(self) -> None:
+        """Reschedule moderators schedule ping."""
+        await self.bot.wait_until_guild_available()
+        schedule_cache = await self.modpings_schedule.to_dict()
+
+        log.info("Scheduling modpings schedule for applicable moderators found in cache.")
+        for mod_id, schedule in schedule_cache:
+            start_timestamp, work_time = schedule.split("|")
+            start = datetime.datetime.fromtimestamp(start_timestamp)
+
+            mod = self.bot.fetch_user(mod_id)
+            self._modpings_scheduler.schedule_at(
+                start,
+                mod_id,
+                self.add_role_schedule(mod, work_time, start)
+            )
+
```

Comment:
This method is full of errors, attach an exception logging done callback to the task to see what needs to be fixed.

Reply:
#1637 was just merged to replace the uses of `self.bot.loop.create_task` that swallow exceptions with `scheduling.create_task(coro, event_loop=bot.loop)` to log exceptions